package/gluon-core/luasrc/lib/gluon/upgrade/220-interface-lan

```diff
@@ -15,6 +15,7 @@ uci:section('network', 'interface', 'mesh_lan', {
 	igmp_snooping = false,
 	proto         = 'gluon_wired',
 	index         = 4,
+	vxlan = site.mesh.vxlan(true),
```

Comment:
indentation of equals.

Reply:
resolved in 8fdc8e70740e38fd6c1946fa9a9fd61d0bdfceed